test/powershell/Modules/Microsoft.PowerShell.Management/Get-ComputerInfo.Tests.ps1

```diff
@@ -341,11 +341,16 @@ public static extern int LCIDToLocaleName(uint localeID, System.Text.StringBuild
         $returnValue = $null
```

Comment:
What about adding 

if (-not (([System.Management.Automation.Platform]::IsNanoServer) -or ([System.Management.Automation.Platform]::IsIoT)))

around the DeviceGuard propertie names within Get-PropertyNamesForComputerInfoTest?